src/app/core/utils/utils.js

```diff
@@ -474,16 +474,27 @@ const utils = {
       })
     }
   },
-  createSingleFieldParameter({field, value, operator='eq', logicop=null}){
-    logicop = logicop && `|${logicop}`;
-    if (Array.isArray(value)){
-      let filter = '';
-      const valueLenght = value.length;
-      value.forEach((value, index) =>{
-        filter+=`${field}|${operator}|${encodeURIComponent(value)}${index < valueLenght - 1 ? `${logicop},` : ''}`
-      });
-      return filter
-    } else return `${field}|${operator.toLowerCase()}|${encodeURIComponent(value)}${logicop || ''}`;
+  createSingleFieldParameter({field, value, operator='eq', logicop=null, search_endpoint="api"}){
```

Comment:
Add new argument 'search_endpoint'. It use to create different filter base on type of search_endpoint value:
api: g3w-admin api
ows: wms filter parameter